adapter/internal/api/restserver/server.go

```diff
@@ -1,6 +1,6 @@
 // Code generated by go-swagger; DO NOT EDIT.
 
-// Copyright (c) 2020, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+// Copyright (c) 2021, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
Why do we change the year of the License header?

Reply:
fixed with https://github.com/wso2/product-microgateway/pull/1826/commits/3aa9b7de891d72fbd25fdfb61f7dac23d0f11d99